contrib/pkg/broker/server/server.go

```diff
@@ -35,15 +36,15 @@ type server struct {
 
 // CreateHandler creates Broker HTTP handler based on an implementation
 // of a controller.Controller interface.
-func CreateHandler(c controller.Controller) http.Handler {
+func createHandler(c controller.Controller) http.Handler {
 	s := server{
 		controller: c,
 	}
 
 	var router = mux.NewRouter()
 
 	router.HandleFunc("/v2/catalog", s.catalog).Methods("GET")
-	router.HandleFunc("/v2/service_instances/{instance_id}", s.getServiceInstance).Methods("GET")
+	router.HandleFunc("/v2/service_instances/{instance_id}/last_operation", s.getServiceInstance).Methods("GET")
```

Comment:
If you change this to be last operation handler, then we should also change the handler to something like s.getLastOperation or something like that to indicate that it's not really the service instance because there's no GET method on that resource.